src/components/text-input/index.md.njk

```diff
@@ -37,7 +37,18 @@ Make text inputs the right size for the content they’re intended for. Postcode
 
 Ensure that users can still easily enter the information they need on smaller screens by snapping text inputs to 100% width at smaller screen sizes.
 
-{{ example({group: 'components', item: 'text-input', frontendComponentName: 'input', example: 'input-width', html: true, nunjucks: true, open: true}) }}
+#### Fixed width inputs
+
+Use fixed width inputs for content that has a specific, known length. These inputs will not re-size until they are was wide as the viewport.
```

Comment:
Change to:

Use fixed width inputs for content that has a specific, known length. The width will remain fixed on all screens unless it is wider than the viewport, in which case it will shrink to fit.